### DISCUSSION TITLE
Hull upgrade "perks" are out of order in mission mode

### DISCUSSION THREAD

**Discussion by NotWendy** [FakeFishGames/Barotrauma#15320] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In the relatively recently added mission mode perk selection menu, the hull upgrade perks are placed out of order and aren't priced appropriately. Physicorium upgrades (normally available for Tier 3 submarines) are cheaper and higher on the list than the titanium-aluminium alloy upgrades (normally available for Tier 2 submarines).
![image](https://github.com/user-attachments/assets/b8a4e23b-1d71-48b1-94d0-487484426fcc)

### Reproduction steps

1. Host a server.
2. Select Mission mode.
3. Pick a submarine and go to the Perk Points menu.
4. Scroll down to the Hull Upgrades category.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/issues/6152 (PR in our private repo), fix coming up in a future unstable build.
